### ISSUE TITLE
Create views for branches and remove them automatically

### ISSUE THREAD

**Issue by nicoddemus** [ESSS/err-jenkins#13]
It would be nice if Bender would automatically create custom views for users when they push a branch. The view would list all jobs which match the name of the branch, so we can have a good look of all jobs related to a branch.

**Comment by tigarmo** [comment]
Good idea, Bender should also remove the custom view when the branch is merged.

**Comment by prusse-martin** [comment]
Can a user create views in other peoples personal views list?

**Comment by nicoddemus** [comment]
> Can a user create views in other peoples personal views list?

I don't think so

**Comment by nicoddemus** [comment]
> Good idea, Bender should also remove the custom view when the branch is merged.

Yep, that's what I meant initially; updated the title to make this more clear. 👍

**Comment by prusse-martin** [comment]
> 
> 
> > Can a user create views in other peoples personal views list?
> 
> I don't think so

I just tested and it looks it is possible.

**Comment by prusse-martin** [comment]
I will sugest to add the created jobs inside "Nested View" to avoid creating excessive polution in the user personal views.
